### PR TITLE
Allow non-block entities to explode

### PR DESCRIPTION
**Pull request by anuar2k**
### Contains

A modification of the ExplosionAuthoritySystem.
onDelayedExplosion event handler required the exploding entities to be a block. Now, with this modification, every entity which does have LocationComponent should be able to explode.

I have also made the DELAYED_EXPLOSION_ACTION_ID public to allow other systems use it, instead of statically defining it again in other places (Minesweeper module is the perfect example).

### CODE REVIEW

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

File: modules/Core/src/main/java/org/terasology/logic/actions/ExplosionAuthoritySystem.java [hunk 1]
```diff
@@ -16,6 +16,8 @@
 package org.terasology.logic.actions;
 
 import com.google.common.collect.Lists;
+import org.slf4j.Logger;
```
Comment (skaldarnar) [inline modules/Core/src/main/java/org/terasology/logic/actions/ExplosionAuthoritySystem.java]: These imports are unused and can be removed.

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by skaldarnar** [comment]
We discussed some further changes to the explosion authority system on slack. Will keep those for later after some more discussion on the interfaces and components that should be available.

**Pull request merged**